cmd/conduit/root/connectorplugins/list.go

```diff
@@ -31,14 +31,20 @@ var (
 	_ ecdysis.CommandWithAliases            = (*ListCommand)(nil)
 	_ ecdysis.CommandWithDocs               = (*ListCommand)(nil)
 	_ ecdysis.CommandWithFlags              = (*ListCommand)(nil)
+	_ ecdysis.CommandWithOutput             = (*ListCommand)(nil)
 )
 
 type ListFlags struct {
 	Name string `long:"name" usage:"name to filter connector plugins by"`
 }
 
 type ListCommand struct {
-	flags ListFlags
+	flags  ListFlags
+	output ecdysis.Output
+}
+
+func (c *ListCommand) Output(output ecdysis.Output) {
+	c.output = output
 }
```

Comment:
Would it make sense to have an `OutputCommand` struct that's embedded, so that the ListCommand can embed it and get the ecdysus.Output field and the Output() method?

Reply:
@hariso hmmm, not entirely sure. I think it'd change how the other decorators behave, and if anything, I'd tackle it in a separate PR to not change its scope.